EIPS/eip-4955.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 4955
+title: Non-Fungible Token Metadata Standard Extension
+description:  A metadata standard extension schema for non-fungible tokens
+author: Ignacio Mazzara (@nachomazzara)
+discussions-to: https://ethereum-magicians.org/t/eip-4955-non-fungible-token-metadata-namespaces-extension/8746
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155
+created: 2022-03-29
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard schema for NFTs metadata.
```

Comment:
I know we say the abstract has to be terse, but this is a bit much :rofl: 

Might want to say something about the exact change ("adding a new field `namespaces` to the JSON schema for NFTs")?